src/Cake.Common/Tools/MSBuild/MSBuildBinaryLogSettings.cs

```diff
@@ -16,17 +16,17 @@ namespace Cake.Common.Tools.MSBuild
     public class MSBuildBinaryLogSettings
```

Comment:
For some reason, I got StyleCop errors for this file so I updated it. I have no clue why...

Reply:
Those errors are correct. However, don't know why they haven't been caught in CI before. Could we possibly keep this PR focused and move the StyleCop issues to a separate PR?

Reply:
I can remove the changes.